### PR TITLE
chore: Add Kotlin for mobile development

### PR DESCRIPTION
**Pull request by shadowsheep1**
- [x] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [x] I've added links to the entry's documentation and/or
      website
- [x] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [x] I've described at least one use case for the entry

### CODE REVIEW

File: radar/languages-and-frameworks/kotlin-mobile.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Kotlin for mobile"
```
Comment (lucacavallaro) [inline radar/languages-and-frameworks/kotlin-mobile.md:2]: I would specify the platform in this entry' title
```suggestion
title: "Kotlin for Android development"
```

**Review by lucacavallaro — APPROVED** [review]

File: radar/languages-and-frameworks/kotlin-mobile.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+quadrant: "languages-and-frameworks"
+tags: [kotlin, mobile]
+---
+
+[Kotlin](https://kotlinlang.org/docs/home.html) is an open-source, statically-typed programming language that supports both object-oriented and functional programming. It is interoperable with the Java virtual machine (JVM), Java Class Libraries and Android.
+
+#### Key features:
+
+1. **Interoperability**: Kotlin interoperates with Java because they compile to the same byte code. It is possible easily call Kotlin code from Java and Java code from Kotlin. This makes adoption much easier and lower-risk. There’s also an automated Java-to-Kotlin converter built into the IDE that simplifies migration of existing code.
+2. **Multiplatform**: Kotlin simplifies the development of cross-platform projects. The Kotlin applications will work on different operating systems like iOS, Android, macOS, Windows, Linux and others. For example, you can create a Kotlin/JS projects with Gradle as a build system.
```
[the remark this answers is not in the document]
Comment (shadowsheep1) [reply radar/languages-and-frameworks/kotlin-mobile.md:13]: Nope, but its a key feature of this language non the less.

**Comment by shadowsheep1** [comment]
We have another React Native module https://github.com/pagopa/io-react-native-jwt that uses Kotlin for the Android native part. We can add it too. What do you think @gunzip @LazyAfternoons ?

**Review by hevelius — APPROVED** [review]
LGTM

**Pull request merged**